src/container/srv_target.c

```diff
@@ -1017,6 +1015,8 @@ cont_close_one_rec(struct cont_tgt_close_rec *rec)
 		hdl->sch_deleted = 1;
 	}
 
+	daos_csummer_destroy(&hdl->sch_csummer);
```

Comment:
this was causing a crash....it concerns me a little that I didn't see it before though